packages/core-typings/src/Apps.ts

```diff
@@ -45,10 +66,11 @@ export type App = {
 	version: string;
 	price: number;
 	purchaseType: string;
-	pricingPlans: PricingPlan[];
+	pricingPlans: AppPricingPlan[];
 	iconFileContent: string;
 	installed?: boolean;
 	isEnterpriseOnly?: boolean;
+	isPurchased?: string;
```

Comment:
Why is this a string?

Reply:
Oversight, will be solving this on the next commit.